tasks/empty.yml

```diff
@@ -1,2 +1,6 @@
-# This file is intentionally left empty and it is used in those Capistrano flow steps
-# where you don't need to execute any custom tasks
\ No newline at end of file
+# This file acts as an informative placeholder in those Capistrano flow steps where
+# you don't currently execute any custom tasks
+---
+- name: ANSISTRANO | DEFAULT_HOOKS | {{ansistrano_hook_name}}
+  local_action: command true
+  changed_when: false
```

Comment:
That would be `ansistrano_verbose_default_hooks` as the parameter is not an imperative statement.